data-explorer/kusto/query/sliding-window-counts-plugin.md

```diff
@@ -22,7 +22,7 @@ Calculates counts and distinct count of values in a sliding window over a lookba
 | *TimelineColumn* | string | &check; | The name of the column representing the timeline.|
 | *Start* | int, long, real, datetime, or timespan | &check; | The analysis start period.|
 | *End* | int, long, real, datetime, or timespan | &check; | The analysis end period.|
-| *LookbackWindow* | int, long, real, datetime, or timespan | &check; | The lookback period. For example, for `dcount` users in past `7d`: *LookbackWindow* = `7d`.|
+| *LookbackWindow* | int, long, real, datetime, or timespan | &check; | The lookback period, this value should be a multiple of the *Bin* value, if not the *LookbackWindow* value will be replaced by a multiple of the *Bin* value. For example, for `dcount` users in past `7d`: *LookbackWindow* = `7d`.|
```

Comment:
```suggestion
| *LookbackWindow* | int, long, real, datetime, or timespan | &check; | The lookback period. This value should be a multiple of the *Bin* value, otherwise the value will be rounded down to a multiple of the *Bin* value. For example, for `dcount` users in past `7d`: *LookbackWindow* = `7d`.|
```